tests/integration/api_v3/http/postTransactions.test.js

```diff
@@ -26,28 +26,27 @@ const {
 } = require('../../../schemas/api_v3/transaction.schema');
 
 const baseUrl = config.SERVICE_ENDPOINT;
-const baseUrlV2 = `${baseUrl}/api/v3`;
-const endpoint = `${baseUrlV2}/transactions`;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/transactions`;
 
-// TODO: Enable test cases once given issue is resolved https://github.com/LiskHQ/lisk-sdk/issues/7172
-xdescribe('Post transactions API', () => {
+describe('Post transactions API', () => {
 	it('Post transaction succesfully', async () => {
 		const postTransaction = await api.post(
-			`${endpoint}?transaction='0802100018002080c2d72f2a200fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a322e0880a0cebdf3ef171214aebd99f07218109162a905d0e0c91e58bedc83c51a0e746f6b656e207472616e736665723a40a30998db4e96911a3d784b0a01b817baf64ec9745d7c67bc537b060eae27ec7ab85ffbb36d76f2b62eaf796deba0407255967506cb01764220e2e6ce66183d07'`,
+			`${endpoint}?transaction=0a05746f6b656e12087472616e7366657218002080c2d72f2a20a3f96c50d0446220ef2f98240898515cbba8155730679ca35326d98dcfb680f032270a0800000000000000001080c2d72f1a1474e3ba5ade3e94451bd4de9d19917c8e6eff624d22003a405fd81dc9d48c04f3a197ea9d57236ff208388a57f7686f19f7917e92679d4da162b97b357f941e3a383f6a795fa9710da5c9dc5eda7007e5b4bbe6551e59360c`,
 		);
 		expect(postTransaction).toMap(postTransactionSchema);
 	});
 
-	it('throws error when posting invalid binary transaction', async () => {
+	it.only('throws error when posting invalid binary transaction', async () => {
```

Comment:
Please do not commit exclusive tests.
```suggestion
	it('throws error when posting invalid binary transaction', async () => {
```

Reply:
WIP commit, kept a pointer for myself to fix.

Reply:
removed.